### PR TITLE
Move to pyteal as pt in ABI tests

### PR DESCRIPTION
**Pull request by michaeldiamant**
Follows #284 to remove temporary flake8 ignores.

Except for `method_return_test.py`, all changes made mechanically via 
[pyteal-as-pt.txt](https://github.com/algorand/pyteal/files/8522722/pyteal-as-pt.txt).  Rename to `.sh` after downloading because Github allows limited upload formats.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -23,15 +23,4 @@ per-file-ignores =
...
-    pyteal/ast/abi/array_dynamic_test.py:           F403, F405
-    pyteal/ast/abi/array_static_test.py:            F403, F405
-    pyteal/ast/abi/bool_test.py:                    F403, F405
-    pyteal/ast/abi/method_return_test.py:           F403, F405
-    pyteal/ast/abi/tuple_test.py:                   F403, F405
-    pyteal/ast/abi/type_test.py:                    F403, F405
-    pyteal/ast/abi/uint_test.py:                    F403, F405
-    pyteal/ast/abi/util_test.py:                    F403, F405
-    # End temporary ignores while merging
+    tests/module_test.py:                           F401, F403
```
Comment (tzaffi) [inline .flake8:26]:
```suggestion
    tests/module_test.py:                           F401, F403

```
nice to have a \n at the end of file

File: pyteal/ast/abi/array_base_test.py [hunk 2]
```diff
@@ -1,48 +1,60 @@
...
-    abi.StaticArrayTypeSpec(abi.Uint32TypeSpec(), 10),
-    abi.StaticArrayTypeSpec(abi.Uint64TypeSpec(), 10),
-    abi.StaticArrayTypeSpec(
-        abi.TupleTypeSpec(abi.BoolTypeSpec(), abi.BoolTypeSpec(), abi.Uint64TypeSpec()),
+import pyteal as pt
+
+options = pt.CompileOptions(version=5)
+
+STATIC_TYPES: List[pt.abi.TypeSpec] = [
+    pt.abi.BoolTypeSpec(),
```
Comment (tzaffi) [inline pyteal/ast/abi/array_base_test.py:9]: 👍 

`pt.abi...` caught me by surprise, but is certainly in the spirit of incoming [CONTRIBUTING.md](https://github.com/algorand/pyteal/blob/d22a93ac9b220c52ab60a21e6292d534283941f1/CONTRIBUTING.md)

File: pyteal/ast/abi/array_dynamic_test.py [hunk 3]
```diff
@@ -1,73 +1,73 @@
 from typing import List
 import pytest
 
-from ... import *
+import pyteal as pt
 from .util import substringForDecoding
```
Comment (tzaffi) [inline pyteal/ast/abi/array_dynamic_test.py:5]: We will need to address these relative imports given the new `CONTRIBUTING.md` ... but that's for a future PR

**Review by tzaffi — APPROVED** [review]
LGTM

**Review by jasonpaulos — APPROVED** [review]

**Comment by michaeldiamant** [comment]
Closing - We opted for the approach in https://github.com/algorand/pyteal/pull/286.

**Pull request closed without merging**